.github/workflows/docker/Dockerfile-dev-ubuntu18.04

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="YiLin Li <YiLin.Li@linux.alibaba.com>"
+
+RUN apt-get update && apt-get install -y make wget git \
+    libseccomp-dev gnupg gcc pkg-config libssl-dev libprotobuf-c0-dev \
+    protobuf-c-compiler binutils-dev debhelper
```

Comment:
protobuf provided by ubuntu 18.04 are not useful for us. We need to build protobuf-c from scratch by self.

In addition, these dependencies are not enough to build shelter from scratch.